explorer/testdata/pointer/print_allocs.carbon

```diff
@@ -0,0 +1,16 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// CHECK:STDOUT: 0: Heap{}, 1: 1, 2: 2
+// CHECK:STDOUT: result: 0
+
+package ExplorerTest api;
+
+fn Main() -> i32 {
+  var a: i32 = 1;
+  var b: i32 = 2;
+  heap.PrintAllocations();
```

Comment:
This test is in `/pointer`, but maybe `/basic_syntax` would be a better home because this doesn't deal with pointers? Similar with `print_allocs_empty`.

Reply:
Ha good, was looking for a good location for that. Done